### PR TITLE
changed clickable div to paper-button so it is accessible

### PR DESCRIPTION
**Pull request by atotic**

### CODE REVIEW

File: iron-doc-viewer.html [hunk 1]
```diff
@@ -56,7 +57,10 @@
 
     <nav id="api">
       <header>API Reference</header>
-      <a id="togglePrivate" on-tap="_togglePrivate">{{_privateToggleLabel}}</a>
+      <paper-button id="togglePrivate"
+        tabindex="0"
```
Comment (cdata) [inline iron-doc-viewer.html]: paper-button will automatically get a tabindex of 0 if you don't specify a tabindex, so you can leave this off.

File: iron-doc-viewer.html [hunk 2]
```diff
@@ -56,7 +57,10 @@
...
+        role="button"
```
Comment (cdata) [inline iron-doc-viewer.html]: paper-button will automatically get role=button if you don't specify a role, so you can leave this off.

**Comment by atotic** [comment]
fixed. PTAL.

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**